### PR TITLE
The first eureka webinar

### PR DESCRIPTION
**Pull request by enrigalmig**

### CODE REVIEW

File: _posts/2016-10-21-The-First-Eureka-Webinar.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title:  The First Eureka Webinar
+date:   2016-10-21 08:00:00
+author: Ezequiel Santiago Sanchez
+categories: Swift, iOS, Eureka
```
Comment (m-revetria) [inline _posts/2016-10-21-The-First-Eureka-Webinar.md]: what do you think about adding `Form`?

Comment (m-revetria) [inline _posts/2016-10-21-The-First-Eureka-Webinar.md]: yup

File: _posts/2016-10-21-The-First-Eureka-Webinar.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+---
+
+[Eureka]:        https://github.com/xmartlabs/Eureka
+[Xmartlabs]:     https://xmartlabs.com/
+<img src="/images/eureka.jpg" alt="Screenshot of hiding rows" />
+
+[Eureka] is Xmartlabs’ elegant form builder for Swift iOS, built and refined for the open source community to accelerate the development cycle and produce cleaner code.
+
+There are over 14k apps currently using Eureka, including Dropbox, Amazon Kindle, SoundCloud, and Foursquare.
```
Comment (enrigalmig) [inline _posts/2016-10-21-The-First-Eureka-Webinar.md]: Maybe links to the app store with _blank?

File: _posts/2016-10-21-The-First-Eureka-Webinar.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+**There are only 30 spots available for this webinar, secure your spot today!**
+
+<a  href="https://goo.gl/forms/rKNRZQ1r5Ay5gM3p2" target="_blank"><img src="/images/save_your_spot.png" alt="Screenshot of hiding rows" /></a>
+
+On the agenda:
+- Overview
+- A demo of Eureka capabilities
+- Eureka’s newest features and add-ons
+- Eureka case studies
```
Comment (enrigalmig) [inline _posts/2016-10-21-The-First-Eureka-Webinar.md]: Eureka community?

**Pull request merged**